### PR TITLE
design FAQ Page

### PR DESCRIPTION
**Pull request by meena-rathi**

### CODE REVIEW

File: src/Hooks/useFAQVisibility.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { useState } from "react";
+
+export const useFAQVisibility = () => {
+  const [showFAQs, setShowFAQs] = useState(false);
+
+  const toggleFAQs = () => {
+    console.log('Toggling FAQ visibility. Current state:', showFAQs); // Debug log
```
Comment (LiskaC) [inline src/Hooks/useFAQVisibility.js:7]: remove console log ^^

File: src/Hooks/useStaatenlosigkeit.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { useState } from "react";
+
+// Ensure this is named correctly
+export const useStaatenlosigkeit = () => {
+  const [showStaatenlosigkeit, setShowStaatenlosigkeit] = useState(false);
+
+  const toggleStaatenlosigkeit = () => {
+    console.log('Toggling Staatenlosigkeit visibility. Current state:', showStaatenlosigkeit);
```
Comment (LiskaC) [inline src/Hooks/useStaatenlosigkeit.js:8]: https://github.com/OleksIva/state_free/pull/17/files#r1748007803

File: src/Hooks/useFAQVisibility.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { useState } from "react";
+
+export const useFAQVisibility = () => {
```
Comment (LiskaC) [inline src/Hooks/useFAQVisibility.js:3]: let's talk about these hooks, I don't think they're necessary

File: src/build/static/js/main.264b971c.js.LICENSE.txt [hunk 4]
```diff
@@ -0,0 +1,82 @@
+/**
```
Comment (LiskaC) [inline src/build/static/js/main.264b971c.js.LICENSE.txt:1]: let's talk about these files, not sure why they're needed

File: src/components/Faqs/FAQItem.jsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (LiskaC) [inline src/components/Faqs/FAQItem.jsx:1]: empty line - remove

File: src/components/Faqs/FAQList.jsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import FAQItem from "./FAQItem";
+import { faqs } from "../../config/Faqs";
+
+const FAQList = ({ isVisible }) => {
+    console.log('FAQList isVisible:', isVisible);
```
Comment (LiskaC) [inline src/components/Faqs/FAQList.jsx:6]: console.log - remove

File: src/components/Faqs/FAQItem.jsx (hunk 5 above)
Comment (meena-rathi) [reply src/components/Faqs/FAQItem.jsx:1]: done

File: src/Hooks/useFAQVisibility.js (hunk 1 above)
Comment (meena-rathi) [reply src/Hooks/useFAQVisibility.js:7]: done

**Pull request merged**